### PR TITLE
Some metrics protocol changes

### PR DESCRIPTION
**Pull request by boks1971**
- Add an envelope time stamp. That should be timestamp when the packet is actually sent.
- Remove `start_timestamp`/`end_timestamp` from `TimeseriesMetric` as it is not clear how it will be used. Can add later if needed.

### CODE REVIEW

**Review by paulwe — APPROVED** [review]

File: protobufs/livekit_metrics.proto [hunk 1]
```diff
@@ -42,22 +43,20 @@ message TimeSeriesMetric {
   uint32 label = 1;
   uint32 participant_identity = 2;
   uint32 track_sid = 3;
-  int64 start_timestamp = 4; // samples
-  int64 end_timestamp = 5;
   // list of samples between start_timestamp and end_timestamp
```
Comment (davidzhao) [inline protobufs/livekit_metrics.proto:46]: update the comment

**Review by davidzhao — APPROVED** [review]

File: protobufs/livekit_metrics.proto (hunk 1 above)
Comment (boks1971) [reply protobufs/livekit_metrics.proto]: oops, cleaning up.

**Review by lukasIO — APPROVED** [review]

**Review by bcherry — APPROVED** [review]

**Comment by davidzhao** [comment]
oh by the way, could you also add a changeset?

**Comment by boks1971** [comment]
I still do not understand how to generate this changeset thing properly. Will read up more. Hopefully, this is good enough for this - https://github.com/livekit/protocol/pull/806/commits/726d0b446d04fe1580d46ecdb7ad2adc51ec9177

**Comment by boks1971** [comment]
> I still do not understand how to generate this changeset thing properly. Will read up more. Hopefully, this is good enough for this - [726d0b4](https://github.com/livekit/protocol/commit/726d0b446d04fe1580d46ecdb7ad2adc51ec9177)

Hmm, I made this `minor`, maybe should just be `patch`?

**Comment by boks1971** [comment]
okay, trying again - https://github.com/livekit/protocol/pull/806/commits/dc586e3cbbcfba7b8e74b85c79128a28c382aa98. Made it a patch. Also, included all packages although `pnpm changeset` said that `@livekit/protocol` is unchanged.

**Comment by davidzhao** [comment]
> okay, trying again - [dc586e3](https://github.com/livekit/protocol/commit/dc586e3cbbcfba7b8e74b85c79128a28c382aa98). Made it a patch. Also, included all packages although `pnpm changeset` said that `@livekit/protocol` is unchanged.

lg!

**Pull request merged**